### ISSUE TITLE
Re-model fee rate coordinator

### ISSUE THREAD

**Issue by bonomat** [get10101/10101#927]
## TL;DR:

Make the routing fee currently hardcoded in the [channel config](https://github.com/get10101/10101/blob/e12d324a241befe8c719afc0f276dfccdc136d92/crates/ln-dlc-node/src/ln/config.rs#L74) configurable from the [settings](https://github.com/get10101/10101/blob/b4e06d514ebb9cd1d2169d8e3c8d5d552d1bd7b1/coordinator/src/settings.rs#L39) and charge the same fee for all channels: 

- when creating new JIT channels
- when opening outbound channels (to non-app users)
- when accepting in-bound channels
- (optional): look into how we can configure routing fee during runtime, document and create a follow-up ticket. 

## Long story

Our node got "exploited" because we were charging 
- 1 sat base fee
- 0% routing fee

Someone used our super cheap routing fee to rebalance their node. 

We should charge a X% routing fee but a 0 sat base fee. This is industry standard as it simplifies path finding. 

We should use the same fee for 
- the routing hint in the interceptable invoice (https://github.com/get10101/10101/blob/a659e992bbbc2054abcfab8c96ea7328a1cd353d/coordinator/src/routes.rs#L144)
- the actual set fee in the channel config (https://github.com/get10101/10101/blob/e12d324a241befe8c719afc0f276dfccdc136d92/crates/ln-dlc-node/src/ln/config.rs#L74)

Right now these two values are not related and we could get weird errors if they diverge. 
The former one needs to be at least as high as the latter one. 

Differently said, since we want this to be configured from the outside, the setting in https://github.com/get10101/10101/blob/b4e06d514ebb9cd1d2169d8e3c8d5d552d1bd7b1/coordinator/src/settings.rs#L39 should be used for all channels (public and private). 

Additionally, it would be good to update routing fees for existing channels. This can be done in a follow-up but might be worth considering.

**Comment by luckysori** [comment]
> Additionally, it would be good to update routing fees for existing channels. This can be done in a follow-up but might be worth considering.

This does not seem to be possible as of today using LDK: https://github.com/lightningdevkit/rust-lightning/blob/df237ba3b455f0ef246604125b8933a7f0074fc5/lightning/src/util/config.rs#L373-L374.

**Comment by luckysori** [comment]
> > Additionally, it would be good to update routing fees for existing channels. This can be done in a follow-up but might be worth considering.
> 
> This does not seem to be possible as of today using LDK: https://github.com/lightningdevkit/rust-lightning/blob/df237ba3b455f0ef246604125b8933a7f0074fc5/lightning/src/util/config.rs#L373-L374.

Perhaps the documentation is outdated, as there is a [`Channel::update_config` method](https://github.com/lightningdevkit/rust-lightning/blob/8e2b70d91a0f7322042d1b0935527c90c59afe45/lightning/src/ln/channel.rs#L1159). I'll create a ticket for this part, as I didn't action it in #951.
